src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -75,10 +80,39 @@ public CommandResult addPersonTags(Model model) throws CommandException {
         personToEdit.setCommonModules(userModuleTags);
 
         model.updateObservablePersonList();
+
+        //
+        // need to take the set of module tags in this command
+        // loop through and then put into the
+        for (ModuleTag tag : moduleTags) {
+            String day = tag.getDay();
+            String start = tag.getStartTime();
+            String end = tag.getEndTime();
+            if (day == null || start == null || end == null) {
+                continue;
+            }
+
+            Module mod = new Module(tag.toString());
+            int startHour = Integer.parseInt(start.substring(0, 2));
```

Comment:
Alright, this works, but the user won’t be able to type in “900” or “9pm” something like that. I think adapting to the second one would be good.

Reply:
I think it is in standard datetime to parse “pm” and “am” so you could look into that.